### PR TITLE
Change getTitle & getTitleTextColor according to current navigation route

### PR DESCRIPTION
**Pull request by kunall17**
Suppose we are in topic narrow and status bar is coloured

simulator screen shot 26-aug-2017 11 51 16 pm
It shows like this on current master 
simulator screen shot 26-aug-2017 11 53 17 pm

Now like this 

simulator screen shot 26-aug-2017 11 51 08 pm

### CODE REVIEW

File: src/title/titleSelectors.js [hunk 1]
```diff
@@ -6,21 +6,27 @@ import { getSubscriptions, getActiveNarrow } from '../selectors';
...
 
 export const getTitleTextColor = createSelector(
   getTitleBackgroundColor,
   getActiveNarrow,
-  (backgroundColor, narrow) =>
-    backgroundColor && (isStreamNarrow(narrow) || isTopicNarrow(narrow))
+  getNav,
+  (backgroundColor, narrow, nav) =>
+    backgroundColor &&
+    nav.routes[nav.routes.length - 1].routeName === 'main' &&
```
Comment (borisyankov) [inline src/title/titleSelectors.js]: Here too.

File: src/title/titleSelectors.js [hunk 2]
```diff
@@ -6,21 +6,27 @@ import { getSubscriptions, getActiveNarrow } from '../selectors';
...
+import { getNav } from '../baseSelectors';
 
 export const getTitleBackgroundColor = createSelector(
   getActiveNarrow,
   getSubscriptions,
-  (narrow, subscriptions) =>
-    isStreamNarrow(narrow) || isTopicNarrow(narrow)
+  getNav,
+  (narrow, subscriptions, nav) =>
+    nav.routes[nav.routes.length - 1].routeName === 'main' &&
```
Comment (borisyankov) [inline src/title/titleSelectors.js]: We already have a selector called `getCurrentRoute` that returns that data.

**Comment by kunall17** [comment]
@borisyankov done 👍

**Comment by borisyankov** [comment]
Thanks. Looks good.

**Pull request merged**